cuckoo.py

```diff
@@ -24,6 +24,7 @@
     from lib.cuckoo.core.startup import init_logging, init_modules
     from lib.cuckoo.core.startup import init_tasks, init_yara, init_binaries
     from lib.cuckoo.core.startup import init_rooter, init_routing
+    from modules.cuckooml.cuckooml import init_cuckooml
```

Comment:
We should move `cuckooml.py` to the `modules/processing/` directory altogether, as it is processing information on a per analysis basis.